substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -323,12 +323,27 @@ pub mod pallet {
 
 	/// Where the reward payment should be made. Keyed by stash.
 	///
+	/// NOTE: Being lazily migrated and deprecated in favour of `Payees`.
+	/// Tracking at <https://github.com/paritytech/substrate/issues/14438>
 	/// TWOX-NOTE: SAFE since `AccountId` is a secure hash.
 	#[pallet::storage]
 	#[pallet::getter(fn payee)]
 	pub type Payee<T: Config> =
```

Comment:
You can rename this and it getter to something like `DeprecatedPayee` and then put a custom `#[storage_prefix = "Payee"]` on it. This will make it easier to audit that this is not being used anywhere.